function.py

```diff
@@ -0,0 +1,6 @@
+
+def function(n):
+    if n == 0:
+        return 1
+    else:
+        return n * function(n-1)
```

Comment:
We may remove the else here
```suggestion
    return n * function(n-1)
```